packages/init/init.ts

```diff
@@ -42,11 +42,11 @@ export default function runTransform(webpackProperties: IWebpackProperties, acti
 			.keys(webpackProperties)
 			.filter((p: string): boolean => p !== "configFile" && p !== "configPath");
 
-	const initActionNotDefined: boolean = (action && action !== "init") || false;
+	const initActionNotDefined = (action && action !== "init") || false;
 
 	webpackConfig.forEach((scaffoldPiece: string): Promise<void> => {
 		const config: IConfiguration = webpackProperties[scaffoldPiece];
-		const transformations: string[] = mapOptionsToTransform(config);
+		const transformations = mapOptionsToTransform(config);
```

Comment:
It doesn't look obvious to me what's the return type of `mapOptionsToTransform`

Reply:
My inspiration for this and the above one is from the Typescript core repo. Again here the type is obvious if you know the method signature or the variable and we don't need to be verbose.

But well, I agree it is highly subjective...

Reply:
Actually there's no need to assign a type if the function already has a signature (and just one, where it returns always a string).